### ISSUE TITLE
Some ad still showing in YouTube RVX Lite app

### ISSUE THREAD

**Issue by vedantdadurwade** [inotia00/revanced-patches#36]
### Type

Other

### Bug description

Ad external links like - "Visit site" is still showing in the YouTube app on home page and below video player.
My version is Youtube ReVanced Extended [v17.41.36][Non-Root] - latest patch
and app architecture is Arm64-v8a , original black theme.

### Steps to reproduce

Simply just open YouTube app

### Relevant log output

```shell
Ads can be seen like - "Visit site", on home page and below video player.
```

### Screenshots or videos

![WhatsApp Image 2022-10-27 at 3 29 37 PM](https://user-images.githubusercontent.com/85695138/198255062-9117baf9-b2cc-4627-880b-91f955dcd91f.jpeg)
![WhatsApp Image 2022-10-27 at 3 29 18 PM](https://user-images.githubusercontent.com/85695138/198255070-5ce3c206-6453-4b50-8c36-fbe14cb630a8.jpeg)
![WhatsApp Image 2022-10-27 at 3 29 04 PM](https://user-images.githubusercontent.com/85695138/198255072-77069b5b-5aba-4b42-830b-fb13433ce14c.jpeg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by inotia00** [comment]
Wrong place to report the issue
And this issue has already been fixed in revanced-patches v2.85.2

**Comment by vedantdadurwade** [comment]
Sorry for reporting at wrong place, btw Thank You for fixing.
